astropy/convolution/convolve.py

```diff
@@ -535,25 +531,24 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
     array([ 1.,  2.,  3.])
```

Comment:
```suggestion
    array([1.,  2.,  3.])
```

Reply:
There's an extra space here and elsewhere in the examples

Reply:
Fixed in 8632901